src/Kernel/Protocol.class.st

```diff
@@ -1,17 +1,32 @@
 "
-A Protocol is a simple value holder representing a protocol.
-It's composed of a name and a set of method selectors
+A Protocol is a cetagorization of methods in Pharo. 
+
+A class has methods, and the methods are classified into protocols.
+
+A class holds my instances (for now through ClassOrganization but we want to remove this class in the future) and I hold the selectors of the methods that are inside me.
+I have a name that is used in browsers to display the different protocols.
 "
 Class {
 	#name : #Protocol,
-	#superclass : #AbstractProtocol,
+	#superclass : #Object,
 	#instVars : [
 		'name',
 		'methodSelectors'
 	],
 	#category : #'Kernel-Protocols'
 }
 
+{ #category : #'instance creation' }
+Protocol class >> ambiguous [
+	^ #ambiguous
```

Comment:
???

Reply:
It is on my TODO list to check what uses that and probably kill that because I don't like it. 

But it's for another PR :)